### PR TITLE
Can_save_changes test correction

### PR DESCRIPTION
**Pull request by ralmsdeveloper**
@anpete  Can_save_changes test correction!

Solved, and @AndriySvyryd had already caught my attention before, I had switched to testing due to locality.

### CODE REVIEW

File: samples/OracleProvider/test/OracleProvider.FunctionalTests/OracleEndToEndTest.cs [hunk 1]
```diff
@@ -424,13 +424,13 @@ public void Can_enumerate_entity_set()
                     Assert.Equal(EntityState.Unchanged, db.Entry(toAdd).State);
                     Assert.DoesNotContain(toDelete, db.ChangeTracker.Entries().Select(e => e.Entity));
 
-                    Assert.Equal(3, Fixture.TestSqlLoggerFactory.SqlStatements.Count);
+                    Assert.Equal(5, Fixture.TestSqlLoggerFactory.SqlStatements.Count);
                     Assert.Contains("SELECT", Fixture.TestSqlLoggerFactory.SqlStatements[0]);
                     Assert.Contains("SELECT", Fixture.TestSqlLoggerFactory.SqlStatements[1]);
                     Assert.Contains(":p0='" + deletedId, Fixture.TestSqlLoggerFactory.SqlStatements[2]);
                     Assert.Contains("DELETE", Fixture.TestSqlLoggerFactory.SqlStatements[2]);
-                    Assert.Contains("UPDATE", Fixture.TestSqlLoggerFactory.SqlStatements[2]);
```
Comment (anpete) [inline samples/OracleProvider/test/OracleProvider.FunctionalTests/OracleEndToEndTest.cs]: So, no batching is expected?

Comment (ralmsdeveloper) [reply samples/OracleProvider/test/OracleProvider.FunctionalTests/OracleEndToEndTest.cs]: Yes, this PR corrects just that!

In fact, this error occurred when I tested with my locale setting,
There were a few places in the code that contained American dates. So I had to make some adjustments.

Thank you!

Comment (ralmsdeveloper) [reply samples/OracleProvider/test/OracleProvider.FunctionalTests/OracleEndToEndTest.cs]: I did the tests and were satisfactory with this adjustment.

5618 passed, 0 failed, 78 skipped (see 'Error List / Messages'), took 450,90 seconds (xUnit.net 2.3.0 build 3820).

Comment (AndriySvyryd) [reply samples/OracleProvider/test/OracleProvider.FunctionalTests/OracleEndToEndTest.cs]: This is expected. After https://github.com/aspnet/EntityFrameworkCore/commit/a22e4069f528693a9fdb72d9e2e819b14b3503c3 we don't batch less than 4 statements by default.

Comment (ralmsdeveloper) [reply samples/OracleProvider/test/OracleProvider.FunctionalTests/OracleEndToEndTest.cs]: So now, it's correct according to https://github.com/aspnet/EntityFrameworkCore/commit/a22e4069f528693a9fdb72d9e2e819b14b3503c3

Comment (anpete) [reply samples/OracleProvider/test/OracleProvider.FunctionalTests/OracleEndToEndTest.cs]: OK, thanks @ralmsdeveloper @AndriySvyryd

**Pull request merged**